scripts/issue.js

```diff
@@ -0,0 +1,16 @@
+let Issue = require('../node_modules/github-api/dist/components/Issue.js')
+let config = require('../config.json')
+
+// create issue
+let createIssue = (title, body, owner, repo) => {
+  let issueObj = new Issue(owner+'/'+repo, {token: process.env.GITHUB_TOKEN})
+  let issueData = {
+    "title": title,
```

Comment:
You could use shorthand:
```
let issueData = {
  title,
  body
}

Reply:
@anuveyatsu I have written in the same pattern in the `milestone.js`, Can we keep it? Otherwise I will have to refactor `milestone.js` as well

Reply:
Well, I think you could integrate some linter here and it would report about all this kind of small improvements. But yes, for now it is minor.